tests/foreman/ui/test_host.py

```diff
@@ -161,11 +164,10 @@ def setUpClass(cls):
         # Search if Libvirt compute-resource already exists
         # If so, just update its relevant fields otherwise,
         # Create new compute-resource with 'libvirt' provider.
-        resource_url = u'qemu+tcp://{0}:16509/system'.format(
-            settings.server.hostname
+        resource_url = u'qemu+ssh://root@{0}/system'.format(
+            settings.compute_resources.libvirt_hostname
```

Comment:
Yeah.. but I think no need of that because validation under `robottelo/settings.py` would do that check.
https://github.com/SatelliteQE/robottelo/blob/master/robottelo/config/settings.py#L353-to-L360L